code/modules/modular_computers/file_system/programs/jobmanagement.dm

```diff
@@ -36,7 +36,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		return FALSE
 	// SKYRAT EDIT ADDITION START
 	if(job.veteran_only)
```

Comment:
one merely cannot create an empty followup to the if condition. this will surely provoke linters.

Reply:
Commented out.